client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
+  border-bottom: 1px solid color('keyline-grey');
+}
+
+.header__upper,
+.header__lower {
+  display: flex;
+}
+
+.header__upper {
+  padding-bottom: 0.8rem;
+  justify-content: flex-end;
+  position: relative;
+
+  @include respond-to('medium') {
+    justify-content: space-between;
+  }
+}
+
+.header__lower {
+  justify-content: space-between;
+  position: relative;
+
+  &.is-active {
```

Comment:
Definitely a question worth considering and deciding on early. I've flip-flopped about whether to use BEM modifiers for state several times.

My current thinking was ultimately cemented by [this post](https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/#6-how-to-handle-states)

> the advantage of the standalone class is that it makes it easy to use JavaScript to apply generic state hooks to any component

Reckons?